app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title page_title
+    base_title = t("rort_sample_app")
+    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
```

Comment:
so sánh giúp a empty?, blank? và present? nhé, trong trường hợp này nên dùng cái nào

Reply:
.nil? và .empty? là hàm của Ruby, .blank? và .present? là các hàm được thêm trong Rails
-----------------------------------
.nil? là một hàm của Object, return true khi đó là nil object.
.empty? chỉ dùng cho object dạng tập hợp (Hash, Array và String), return true nếu là mảng rỗng [], chuỗi chỉ gồm khoảng trắng và hash rỗng {}.
.blank? dùng cho bất kỳ Object nào, return true nếu nó là false hoặc empty
.present? dùng cho bất kỳ Object nào, ngược lại với .blank?, return true nếu object NOT nil, NOT false hoặc NOT empty